docs/src/targene/tmle.md

```diff
@@ -61,3 +63,80 @@ All estimators will learn the nuisance functions `Q` and `G` with the provided `
 For the list of available models and resampling strategies, checkout the [TargetedEstimation documentation](https://targene.github.io/TargetedEstimation.jl/stable/models/).
 
 For full details, on available estimators and how to specify them, visit the [TMLE.jl documentation](https://targene.github.io/TMLE.jl/stable/).
+
+## Current recommanedations
+
+At present, there are two main use cases for Targene: (1) investigating an average treatment effect (ATE) or (2) an interaction average treatment effect (IATE). Here we present our current recommendation fo estimators for these two types of analyses.
+
+1. ATE
+
+```julia
```

Comment:
I thought the suggestion was to use GLMNet. Maybe also briefly explain what is GLMNet, e.g. cross-validated L1 penalized linear model.